src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -30,6 +30,7 @@ IqPolicyEvaluation.FailOnNetwork=Fail build when unable to communicate with IQ S
 IqPolicyEvaluation.JobSpecificCredentials=Use job specific credentials
 
 IqPolicyEvaluation.NoIqServersConfigured=No IQ Server configured.
+IqPolicyEvaluation.ResourceNotFound=Possible version mismatch.  Please check if the IQ Server is compatible with this plugin.
```

Comment:
Since this is implemented for a very specific case, I would like to see it limited to that case, which would allow us to have a very specific message with the actual IQ server version that is required.

We have another Jira for genericmismatches between server and plugin versions .

Reply:
> Since this is implemented for a very specific case, I would like to see it limited to that case, which would allow us to have a very specific message with the actual IQ server version that is required.

Changed 31c1544